src/css/profile/mobile/common/toast.less

```diff
@@ -57,15 +54,39 @@
 					}
 				}
 
-				&::after {
-					border-left: 2 * @unit_base solid @color_toast_divider;
-					height: 48 * @unit_base;
-					content: "";
-					position: absolute;
-					left: -4 * @unit_base;
-					top: 50%;
-					transform: translate(0, -50%);
-				}
+			}
+		}
+	}
+}
+
+@media all and (max-width: 479px) {
```

Comment:
How about moving the @media all and (max-width: 479px) { under the .ui-popup-content like as
```
.ui-popup.ui-popup-toast {
 		.ui-popup-content { 
@media all and (max-width: 479px) {
// Changes
}

@media all and (min-width: 480px) and (max-width: 959px) {
// Changes
}
```

Reply:
Thank you for your comment. It was helpful